app/models/invite.js

```diff
@@ -13,33 +12,18 @@ export default Model.extend({
     updatedAtUTC: attr('moment-utc'),
     updatedBy: attr('number'),
     status: attr('string'),
-    roles: hasMany('role', {
-        embedded: 'always',
-        async: false
-    }),
+    role: belongsTo('role', {async: false}),
```

Comment:
Are roles retrievable from the API via an ID (e.g. `/api/roles/:id/`)? If so then we can remove the `{async: false}` option - it's required for embedded records but when we only get an ID it's safer to go with the default behaviour and allow async retrieval of the associated role model.